### ISSUE TITLE
Crash when set containerWhitespace

### ISSUE THREAD

**Issue by maciek-s** [panpf/zoomimage#63]
**Describe the bug**

App crash when swipe without scroll after set `containerWhitespace`

**Affected platforms**

- Android

**Affected components**

Select of the components below:

- ZoomImage

**Versions**

- zoomimage version*: `1.1.0-rc02`
- Kotlin version*: `2.0.21`
- Compose version(s)* (Jetpack/Multiplatform): `1.7.1`

**Running Devices**

Please accurately describe the device Model, OS version, and CPU Architecture

- MacBookPro m2
- Android emulator

**Sample code**

```
Scaffold(
    topBar = {
        TopAppBar(
            title = {},
            colors = TopAppBarDefaults.topAppBarColors(
                containerColor = MaterialTheme.colorScheme.surface.copy(alpha = 0.8f)
            )
        )
    }
) { innerPadding ->
    val zoomState: ZoomState = rememberZoomState()

    LaunchedEffect(zoomState.zoomable) {
        zoomState.zoomable.apply {
            disabledGestureTypes = disabledGestureTypes or GestureType.ONE_FINGER_SCALE
            containerWhitespace = ContainerWhitespace(vertical = 100f, horizontal = 0f)
        }
    }

    val painter = rememberAsyncImagePainter(
        model = "https://picsum.photos/2000/2000"
    )

    ZoomImage(
        modifier = Modifier.fillMaxSize(),
        alignment = Alignment.TopCenter,
        painter = painter,
        contentDescription = null,
        contentScale = ContentScale.Fit,
        zoomState = zoomState,
        scrollBar = null,
    )
}
```

**Reproduction steps**

When swipe on image without zooming app crashed
```
java.lang.IllegalArgumentException: Cannot coerce value to an empty range: maximum -423.0 is less than minimum 424.0.
                                                                                                    	at kotlin.ranges.RangesKt___RangesKt.coerceIn(_Ranges.kt:1442)
                                                                                                    	at com.github.panpf.zoomimage.compose.util.Compose_platform_utilsKt.limitTo-3MmeM6k(compose_platform_utils.kt:362)
```

**Screenshots**

https://github.com/user-attachments/assets/d2e37340-75af-4c3b-951a-0b026e386eda

**Question**

In my sample, I have a transparent toolbar. Is it possible to display an image beneath the toolbar and allow zooming without clipping to the padding? Unfortunately, `containerWhitespace`, as shown in the video, is added after zooming. Additionally, when I add padding to the `ZoomImage`, it does not appear beneath the toolbar after zooming.

**Comment by panpf** [comment]
The problem has been found and is being solved

**Comment by panpf** [comment]
This issue has been fixed on the dev branch and a snapshot version is being released. You can download version 1.1.0-rc03-SNAPSHOT from the snapshot repository in two hours to test if there is still any problem.

**Comment by maciek-s** [comment]
Thanks, I tested snapshot release, and issue no longer occurs.
How about my question at  bottom of bug description? It is possible to add top padding to `ZoomImage` and when zoom in do not clip image to padding?

**Comment by panpf** [comment]
Open a new issue. This is similar to setting padding in View but clipPadding = false

**Comment by panpf** [comment]
Fixed in version 1.0-rc03
